README.md

```diff
@@ -5,86 +5,113 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/canonical/microceph/microceph)](https://goreportcard.com/report/github.com/canonical/microceph/microceph)
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-<p align="center">
-<a href="https://snapcraft.io/microceph">MicroCeph</a> is snap-deployed Ceph with built-in clustering.
-</p>
-
 [![Get it from the Snap Store][snap-button]][snap-microceph]
 
-## Table of contents
 
-* [💡 Philosophy](#-philosophy)
-* [🎯 Features](#-features)
-* [⚡️ Quickstart](#%EF%B8%8Fquickstart)
-* [📖 Documentation](#-documentation)
-* [💫 Project & community](#-project--community)
-* [📰 License](#-license)
+MicroCeph is an opinionated command-line tool for deploying and managing Ceph clusters at all scales.
+It reduces deployment and management overhead by simplifying key distribution, service placement,
+and disk administration through a single interface.
+
+Available as a snap, MicroCeph is the easiest tool for admins, developers, and hobbyists to manage clusters.
+
+## Installation
+
+MicroCeph has first-class support as a snap. On snap-ready systems, you can install it on the command line with:
+
+```
+sudo snap install microceph
+```
+
+Disable automatic snap upgrades, so that no unexpected updates change your set-up:
+
+```
+sudo snap refresh --hold microceph
+```
+
+## Basic usage
+
+MicroCeph can deploy a Ceph cluster on a single machine with minimal commands.
+
+First, set up a Ceph cluster on your machine with:
+
+```
+sudo microceph cluster bootstrap
+```
+
+> [!NOTE]  
+> `cluster` is a microceph subcommand for managing clusters.
+
+After setup, add storage to your cluster with:
+
+```
+sudo microceph disk add loop,4G,3
+```
+
+Here, you’ll add three virtual disks (“loop file” disks) of 4 GiB each.
+
+
+Once your cluster is set up and running, you can monitor its status with:
+
+```
+sudo microceph status
+```
+
+Note that there are no spaces between the `disk add` arguments.
 
-## 💡 Philosophy
 
-Deploying and operating a Ceph cluster is complex because Ceph is designed to
-be a general-purpose storage solution. This is a significant overhead for small
-Ceph clusters. MicroCeph solves this by being _opinionated_ and _focused_ on
-the small scale. With MicroCeph, deploying and operating a Ceph cluster is as
-easy as a [Snap][snap-microceph]!
+If you need a comprehensive status report of your cluster, including its health and disk usage, run:
 
-## 🎯 Features
+```
+sudo microceph.ceph status
+```
```

Comment:
we promote direct usage of ceph commands like `sudo ceph status` so maybe mention that (we can use ceph commands!) and guide user to check ceph status ?

Reply:
Good point.